packages/x-charts/src/internals/stacking/offset/offsetDiverging.test.ts

```diff
@@ -0,0 +1,349 @@
+import { expect } from 'vitest';
+// import { offsetDiverging } from './offsetDiverging';
+import { stackOffsetDiverging as offsetDiverging } from '@mui/x-charts-vendor/d3-shape';
```

Comment:
I assume the goal is to test our version, not the d3 code :)

```suggestion
import { offsetDiverging } from './offsetDiverging';
```

Reply:
should be better now, also simplified generating the series for testing 😅